jina/executors/indexers/keyvalue.py

```diff
@@ -28,7 +28,7 @@ class ReadHandler:
         def __init__(self, path):
             with open(path + '.head', 'rb') as fp:
                 tmp = np.frombuffer(fp.read(), dtype=np.int64).reshape([-1, 4])
-                self.header = {r[0]: r[1:] for r in tmp}
+                self.header = {r[0]: None if np.array_equal(tmp[0][1:], (-1, -1, -1)) else r[1:] for r in tmp}
```

Comment:
My comments got deleted somehow, writing again:

Cant you just rewrite the entire header? Much smaller than body, shouldnt be a perf hindrance.

Or at least make `(-1, ...)` a constant

Reply:
Now, I created a constant for `(-1, -1, -1)`.